tests/unit/App.spec.ts

```diff
@@ -1098,7 +1091,7 @@ describe('Restoration - App page', () => {
   it('renders the sub-components properly', () => {
     expect(wrapper.findComponent(SbcHeader).exists()).toBe(true)
     expect(wrapper.findComponent(SbcFooter).exists()).toBe(true)
-    expect(wrapper.findComponent(SbcFeeSummary).exists()).toBe(true)
+    // expect(wrapper.findComponent(SbcFeeSummary).exists()).toBe(true)
```

Comment:
any particular reason this is commented out?

Reply:
Yup, there are a couple of tests that are failing on certain lines. This is one of them. I still either need to fix it or maybe create another ticket to handle the remaining tests.

Reply:
No worries, just trying to help out.. it looks pretty good so far

Reply:
Whoops, I added some reviewers 🤦

Reply:
This test is fixed so I'm resolving this conversation.